### PR TITLE
Add python hello world program

### PR DESCRIPTION
**Pull request by MarshallAsch**
**Why**

This was added as an example to show how a contribution can be made to the repository. 

---

**What**

This adds a `hello world` program written in python. 

---

**Verification**

When you run `./submissions/hello_world/HelloWorld.py` the output should be `Hello, World!`

---

**Checklist:**

- [x] This contribution abides by all rules

### CODE REVIEW

File: submissions/hello_world/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+$ cd submissions/hello_world
+```
+
+```bash
+$ ./HelloWorld.py
+```
+
+## Why I Added it
+
+This program is being added as an example contribution.
```
Comment (SoorajModi) [inline submissions/hello_world/README.md:22]: You should add a list of contributors in this README

File: submissions/hello_world/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Hello World
+
+This is an example hello world program.
+It is s simple executable python3 script that will print `Hello, World!` when run.
```
Comment (SoorajModi) [inline submissions/hello_world/README.md]: Nit: typo, the s should be an a

**Review by SoorajModi — CHANGES_REQUESTED** [review]

**Comment by SoorajModi** [comment]
Nit: can we rename the PR to `Add hello world program`

File: submissions/hello_world/README.md (hunk 2 above)
Comment (MarshallAsch) [reply submissions/hello_world/README.md]: fixed in a592764

File: submissions/hello_world/README.md (hunk 1 above)
Comment (MarshallAsch) [reply submissions/hello_world/README.md:22]: fixed in ceeb583

Comment (MarshallAsch) [reply submissions/hello_world/README.md:22]: fixed in [ceeb583](https://github.com/SoorajModi/ContributeAnythingGryphHacks23/pull/1/commits/ceeb5839ab47588ff1771445c395f9c3cd5270fb)

**Review by SoorajModi — APPROVED** [review]

**Pull request merged**